server/index.js

```diff
@@ -58,6 +81,88 @@ app.get('/gastos', (req, res) => {
   })
 })
 
+// VISTA
+app.get('/test', (req, res) => {
+  res.send({
+    hola: 'hola',
+    cantidad: 2
+  });
+});
+
+app.post('/suma', (req, res) => {
+  const body = req.body;
+  const suma = body.cantidad1 + body.cantidad2;
+  
+  console.log(suma);
+  // responder la suma de dos numeros 
+  res.send(String(suma))
+});
+
+app.post('/registro', (req, res) => {
+  const body = req.body;
+  const email = body.email;
+  const password = body.password;
+  
+  console.log({
+    email, 
+    password
+  });
+
+  // SQL Guardar en base de datos
+
+  // responder la suma de dos numeros 
+  res.send({ respuesta: `El registro fue exitoso del usuario ${email} fue existoso.` });
+});
+
+// Renderizar el archivo login.hbs
+app.get('/login', (req, res) => {
+  session = req.session;
+  //console.log(session.userid);
+  //como if para verificar si hay session activa 
+  if(session.userid){
+    res.send("Welcome User <a href=\'/logout'>click to logout</a>");
+  }
+  else{
+    res.render('login', {});
+  }
+ 
+});
+
+// Traer los datos del HTML form de login.hbs en el body del request (req.body)
+app.post('/user', (req, res) => {
+//se trago del body de html (username y password)
+  const body = req.body;
+  const username = body.username;
+  const password = body.password;
+
+  //console.log('HHHHHH',username, password)
+
+   client.query(`SELECT (name,password) FROM users WHERE name='${username}' and password='${password}';`, (err, respuesta) => {
```

Comment:
Creo que es mejor comparar por email. Y no buscar por password porque el password se guarda encriptado. 

Obviamente requiere que recibas diferentes parametros en el body. 🙂
```suggestion
   client.query(`SELECT (name,password) FROM users WHERE email='${email}';`, (err, respuesta) => {
```

Reply:
LA NETA NO JAJAJAJA

Reply:
ya lo cambiamos por email